packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -331,6 +341,18 @@ export class SchemaPublisher {
 
     let checkResult: SchemaCheckResult;
 
+    const approvedSchemaChanges = new Map<string, SchemaChangeType>();
+
+    if (contextId !== null) {
```

Comment:
do we check if `contextId` is an empty string (or string with whitespaces only)?

Reply:
Good catch in general we should do some `contextId` validation to avoid someone using a novel as the context id

Reply:
https://github.com/kamilkisiela/graphql-hive/pull/3359/commits/66ca884fe98c59867d15fd26ca165c7ceddd0d26